modules/hal_nxp/mcux/mcux-sdk/CMakeLists.txt

```diff
@@ -89,6 +89,10 @@ if (DEFINED CONFIG_SOC_RESET_HOOK OR DEFINED CONFIG_SOC_SERIES_IMXRT6XX
     include(device_system_MCXN947_cm33_core0)
   elseif (CONFIG_SOC_MCXN947_CPU1)
     include(device_system_MCXN947_cm33_core1)
+  elseif (CONFIG_SOC_MCXN547_CPU0)
+    include(device_system_MCXN547_cm33_core0)
+  elseif (CONFIG_SOC_MCXN547_CPU1)
```

Comment:
no space between if/elseif and bracket in cmake

Reply:
Done.